docker-compose-integration-test.yml

```diff
@@ -37,13 +38,13 @@ services:
   ping-server:
     build: experimenter/tests/integration/nimbus/utils/ping_server
     ports:
-      - "5000:5000"
+      - "5050:5050"
```

Comment:
5000 no good? 🤔

Reply:
I was getting port used on my machine everytime so I switched it

Reply:
It's a mac thing, something is broadcasting on port 5000

Reply:
Aaaaaahhhhh